src/main/kotlin/racing/model/Name.kt

```diff
@@ -0,0 +1,18 @@
+package racing.model
+
+data class Name(
```

Comment:
값이 하나뿐인 일급객체는 다음과 같이 표현해도 좋아요. 

```suggestion
@JvmInline
value class Name(
```